superset/migrations/versions/8d49a37823bf_remove_tables_uniqueness.py

```diff
@@ -0,0 +1,79 @@
+"""remove_tables_uniqueness
+
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Revision ID: 8d49a37823bf
+Revises: 18dc26817ad2
+Create Date: 2019-01-20 11:44:14.640628
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '8d49a37823bf'
+down_revision = '18dc26817ad2'
+
+from alembic import op
+import sqlalchemy as sa
+import pprint
+
+from superset.utils.core import generic_find_uq_constraint_name
+from collections import OrderedDict
+
+def is_unique_constraint(constraint):
+    return constraint and isinstance(constraint, sa.UniqueConstraint)
+
+def is_sqlite():
+    bind = op.get_bind()
+    print(f'The bind name is {bind.dialect.name}')
+    return bind and bind.dialect and bind.dialect.name and bind.dialect.name.startswith('sqlite')
+
+def upgrade():
+    bind = op.get_bind()
+    insp = sa.engine.reflection.Inspector.from_engine(bind)
+    constraints = insp.get_unique_constraints('tables')
+    print(f'Printing the existing uniq constraints in table {constraints}')
+    if not constraints:
+        constraints = []
+    if is_sqlite():
```

Comment:
This hacking of is-sqlite checking was necessary to force it to drop the unique constraint. In the absence of this (even with batch_alter_table, recreate=always) it wasn't really creating a new copy of the table with the uniq constraint removed. 

This is a bit hacky for sure, but it works for mysql, sqlite and postgresql. Further, it only pays the recreate cost when needed. 

I think it would be a bit premature to refactor this and move into utils, but I would be happy to put that there just as well.